src/components/QuickOpenModal.js

```diff
@@ -215,27 +225,20 @@ export class QuickOpenModal extends Component<Props, State> {
   };
 
   onKeyDown = (e: SyntheticKeyboardEvent<HTMLElement>) => {
-    const { selectLocation, selectedSource, enabled, query } = this.props;
+    const { enabled } = this.props;
     const { results, selectedIndex } = this.state;
 
     if (!enabled || !results) {
       return;
     }
 
     const canTraverse = !this.isGotoQuery();
-    if (e.key === "ArrowUp" && canTraverse) {
-      return this.traverseResults(-1);
-    } else if (e.key === "ArrowDown" && canTraverse) {
-      return this.traverseResults(1);
+    const isKeyUpDown = e.key === "ArrowUp" || e.key === "ArrowDown";
+    if (canTraverse && isKeyUpDown) {
+      return this.traverseResults(e.key === "ArrowUp" ? -1 : 1);
     } else if (e.key === "Enter") {
```

Comment:
we can replace `else if` with `if` because each case is returning early when hit